src/cfnlint/data/ExtendedSpecs/append_spec/allowed_values.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Is this just a place holder file for now?

Reply:
Yes. Need to merge into the property Values one with the valuetypes